src/vm/unixlowmem.cpp

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ===========================================================================
+
+#include "common.h"
+#include "unixlowmem.h"
+#include <stdlib.h>
+#include "debugmacros.h"
+
+#ifdef FEATURE_PAL
```

Comment:
This is not necessary as we add the source to the list of files we build for Unix only.